apps/coscrad-frontend/src/components/header/header.tsx

```diff
@@ -1,21 +1,40 @@
+import styled from '@emotion/styled';
+import { AppBar, Toolbar, Typography } from '@mui/material';
+import { Box } from '@mui/system';
 import { useContext } from 'react';
 import { Link } from 'react-router-dom';
 import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
-import { NavBar } from '../nav-bar/nav-bar';
-import './header.css';
+import { NavMenu } from '../nav-menu/nav-menu';
+
+const StyledAppBar = styled(AppBar)({
```

Comment:
I like the wrapper approach, but is it the header's job to do this wrapping? Maybe there's a cleaner pattern?

Reply:
Have to look into that.  In this case, `AppBar` is just the coloured bar at the top of the screen with the nav menu and site title, so it does seem to me to be consistent with what a Header does.  We can definitely break out the `styled` components into separate files.

Reply:
Ok. This file isn't overly complicated, so let's leave it  be for now.